### DISCUSSION TITLE
Which release will include the new List/Read Certificate methods?

### DISCUSSION THREAD

**Discussion by rageshkrishna** [rajanadar/VaultSharp#219] [General]
I'm really looking forward to using the `ListCertificatesAsync` and `ReadCertificateAsync` methods that got added recently. I was wondering if there are any plans for a release in the near future that will include this functionality. Thanks!

**Comment by rajanadar** [comment]
hi @rageshkrishna 

It's planned for this week. Give it a couple more days.

**Comment by rageshkrishna** [comment]
That's fantastic! Thanks for the update.

**Comment by rajanadar** [comment]
@rageshkrishna  The 1.7.0.1 version is now published. Let me know if any issues.
